content/influxdb/clustered/admin/bypass-identity-provider.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Bypass your identity provider
+description: >
+  InfluxDB clustered generates a valid management token (known as the _admin token_)
+  that can be used in development and testing environments in lieu of configuring
+  and using an OAuth2 identity provider.
+menu:
+  influxdb_clustered:
+    parent: Administer InfluxDB Clustered
+weight: 207
+---
+
+{{< product-name >}} generates a valid [management token](/influxdb/clustered/admin/tokens/#management-tokens)
```

Comment:
I think we should avoid calling the admin token a "management token" and referring to the management tokens documentation, because the functionality differs pretty notably. If a customer lists their management tokens, they won't see the admin token listed, and they also have to revoke the admin token in a completely different way.

I do think there's value in generating a management token as the admin token, but that isn't what we currently do, and we haven't prioritized that work.

Reply:
Sounds good. I'll update this verbiage.